MatterControl.Printing/Settings/SliceSettingsFields.cs

```diff
@@ -2117,7 +2117,7 @@ public static IEnumerable<SliceSettingData> AllSettings()
 				{
 					SlicerConfigName = SettingsKey.z_offset,
 					PresentationName = "Z Offset".Localize(),
-					HelpText = "The distance to move the nozzle along the Z axis to ensure that it is the correct distance from the print bed. A positive number will raise the nozzle, and a negative number will lower it.".Localize(),
+					HelpText = "DEPRECATED: replaced with extruder_offset".Localize(),
```

Comment:
@larsbrubaker - where does this have an effect? I'd expect that we removed the old field and and removed this one from layouts?